bisqapps/shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/data/model/Settings.kt

```diff
@@ -1,5 +1,8 @@
 package network.bisq.mobile.domain.data.model
 
+import kotlinx.serialization.Serializable
+
+@Serializable
```

Comment:
Not form your change, but I would rename `bisqUrl` to a more clear name. Like `bisqApiUrl`

Reply:
might be a good opportunity to do so, I'll have a look